src/ngraph_api.cc

```diff
@@ -45,87 +47,61 @@ bool ngraph_list_backends(char** backends, int backends_len) {
   return true;
 }
 
-bool ngraph_set_backend(const char* backend) {
-  if (SetBackend(string(backend)) != tensorflow::Status::OK()) {
+bool NgraphSetBackend(const char* backend) {
```

Comment:
We might want C++ functions for SetBackend, ListBackends, NgraphGetCurrentlySetBackendName etc.
Since we need C++ config api's too.